components/network/ntpsec/files/ntpsec

```diff
@@ -22,14 +22,16 @@
 
 #
 # Copyright (c) 2009, 2011, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2016 Andrew Stormont <andyjstormont@gmail.com>
+# Copyright 2017 OmniOS Community Edition (OmniOSce) Association.
 #
 
-# Standard prolog
+# Standard prologue
 #
 . /lib/svc/share/smf_include.sh
 
 if [ -z $SMF_FMRI ]; then
-        echo "SMF framework variables are not initialized."
+        echo "SMF framework variables are not initialised."
```

Comment:
Don't do it, it's English/American English mess.  :)

Reply:
That came from the British section of OmniOS :).